File: database4.py
# select the details on the basis of where clause
import mariadb
import sys
try:
           conn=mariadb.connect(
                      user="root",
                      password="maria",
                      host="127.0.0.1",
                      port =3308,
                      database="dhiraj"
                      )
except mariadb.Error as e:
           print("Error to connect",e)
           sys.exit(1)

print("Database connected successfully :)")
# get cursor
cur=conn.cursor()
try:
    cur.execute("select name,age from student where id>104")
    # select name , age from student where id is greater than 104
except mariadb.Error as e:
    print("Error",e)
for i,j in cur:
    print("name",i)
    print("age",j)

conn.close()

File: database4.js
//insert value into the table 
var mysql=require("mysql")

var conn= mysql.createConnection({host:"localhost",user:"root",password:"rootpassword",database:"node"})

conn.connect(function(err){
    if(err ) throw err;
    console.log("connected :)");

var sql = "insert into  nodetable values('Dhiraj', 'Bhiwadi',1)"; //insert single record into the table
conn.query(sql,function(err,result){ 
    if(err ) throw err;
    console.log("one record inserted  :)");
});
});
